docs/guides/component-testing/angular/api.mdx

```diff
@@ -11,6 +11,15 @@ sidebar_position: 40
 import { mount } from 'cypress/angular'
 ```
 
+:::info
+Be sure to use the `cypress/angular-signals` package if using Angular `17.2` and up and wishing to test `signal()`s within your component tests.
+
+```js
+import { mount } from 'cypress/angular-signals'
+```
+
+:::
```

Comment:
Don't think this needs to be wrapped in an info here. Kind of looks weird.

![Screenshot 2024-06-11 at 10 08 05 AM](https://github.com/cypress-io/cypress-documentation/assets/1271364/c0608d7a-a26a-4eec-af5c-98591d43e02c)

```suggestion
Be sure to use the `cypress/angular-signals` package if using Angular `17.2` and up and wishing to test `signal()`s within your component tests.

```js
import { mount } from 'cypress/angular-signals'
```
```

Reply:
addressed in 038e1b240